src/hydro_system.hpp

```diff
@@ -204,7 +213,8 @@ template <typename problem_t> auto HydroSystem<problem_t>::maxSignalSpeedLocal(a
 					} else {
 						const auto Etot = cons[bx](i, j, k, HydroSystem<problem_t>::energy_index);
 						const auto Eint = Etot - kinetic_energy;
-						const auto P = Eint * (HydroSystem<problem_t>::gamma_ - 1.0);
+						const auto P =
+						    ComputePressure(cons[bx], i, j, k); // const auto P =  Eint * (HydroSystem<problem_t>::gamma_ - 1.0);
```

Comment:
@BenWibking my modification in this line returns a NAN for the pressure. It could be that the `cons` array the ComputePressure function is expecting is not the same as the `cons[bx]` array being used here.

Reply:
Do you know why this would give a NAN for pressure

Reply:
I don't see an obvious error. Might have to just step through it in a debugger.

Reply:
Ugh, it was the mean molecular weight being NAN for the test where this error was coming from.